cloudformation/elasticsearch.yaml

```diff
@@ -47,7 +47,7 @@ Resources:
     Type: AWS::IAM::Role
     Condition: isDev
     Properties:
-      RoleName: !Join ['-', ['kibana-cognito-role', !Ref AWS::Region]]
+      RoleName: !Join ['-', ['kibana-cognito-role', !Ref Stage, !Ref AWS::Region]]
```

Comment:
Any particular reason for Stage and then Region as opposed to Region and then Stage?